src/nestjs-back/src/utils/checkPattern.ts

```diff
@@ -0,0 +1,19 @@
+import { UnauthorizedException } from '@nestjs/common';
+
+/*
+ * Utils for pattern validation
+ */
+
+export const checkPasswordPattern = (password: string) => {
```

Comment:
To be incorporated in its own validator
Remove the throwing of the HTTP exception then